wp-pgp-encrypted-emails.php

```diff
@@ -1251,14 +1204,13 @@ private static function prepareMail ($to, $subject, $message, $headers, $attachm
         $pub_cert      = false;
         $erase_subject = false;
 
-        // Note: If admin email is also a user and key only exists with user, this won't find it.
         if (get_option('admin_email') === $to) {
-            if (get_option(self::$meta_key_encryption_type)==1) $pub_key = self::getAdminKey();
-            else if (get_option(self::$meta_key_encryption_type)==2) $pub_cert = self::getAdminCert();
+            if (get_option(self::$meta_key_encryption_type)=='pgp') $pub_key = self::getAdminKey();
```

Comment:
Also a somewhat minor thing, but there are numerous style things that would make this clearer and safer:

* Use a [Yoda condition](https://en.wikipedia.org/wiki/Yoda_conditions) in these statements: instead of `$variable == 'literal'`, use `'literal' === $variable`.
* Use braces even with single-statement `if`/`else` blocks: `if (condition) { statement; }`.
* Alternatively, this sort of statement is a lot more concise with a `switch`.
* You're also using either `$pub_key` or `$pub_cert` and taking different code paths depending on the presence of one or the other of this variable later on, but you already created an explicit toggle; you should consider what it would mean to use that here, as well.

This is also why I'm not totally sure the toggle is the way to do this, but I don't have a better suggestion at the moment and I'm also totally drunk and hungover and probably shouldn't even be reviewing this code right now, lulz.